source/bricklaunch

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
+#
+# bricklaunch.sh <ops> <program>
+# - open a new vt and switch to it
+# - give 'user' ownership of the vt
+# - run the given programm on the new vt as 'user'
+#
+
+OPTIND=1
+user='user'
+openvt_opts=""
+rm_ttyfile=0
+
+function show_help {
+  cat <<END_OF_HELP
+bricklaunch.sh <opts> <program>
+
+  -h        this help
+  -s        switch to the new virtual terminal
+  -t <file> file to write the path of the new tty to
+  -u <user> user to run the program under
+  -w        wait for the program to finish
+END_OF_HELP
```

Comment:
Sounds useful. I also have some ideas for the script, e.g. it would be nice to forward the exit-status from the program. Maybe it is already enough to move the rm on the last line to an atexit handler.